examples/DynamicProperties-sample.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8"?>
+<edmx:Edmx Version="4.0" xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx">
+  <edmx:Reference Uri="https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Aggregation.V1.xml">
+    <edmx:Include Namespace="Org.OData.Aggregation.V1" Alias="Aggregation" />
+  </edmx:Reference>
+  <edmx:Reference Uri="https://sap.github.io/odata-vocabularies/vocabularies/UI.xml">
+    <edmx:Include Namespace="com.sap.vocabularies.UI.v1" Alias="UI" />
+  </edmx:Reference>
+  <edmx:Reference Uri="https://sap.github.io/odata-vocabularies/vocabularies/Analytics.xml">
+    <edmx:Include Namespace="com.sap.vocabularies.Analytics.v1" Alias="Analytics" />
+  </edmx:Reference>
+  <edmx:DataServices>
+    <Schema Namespace="DynamicProperties.examples" Alias="self" xmlns="http://docs.oasis-open.org/odata/ns/edm">
+
+      <EntityType Name="Sales">
+        <Key>
+          <PropertyRef Name="Order"/>
+        </Key>
+        <Property Name="Order" Type="Edm.String" Nullable="false"/>
+        <Property Name="Country" Type="Edm.String" Nullable="false"/>
+        <Property Name="NumberOfItems" Type="Edm.Int16" Nullable="false"/>
+        <Property Name="Amount" Type="Edm.Decimal"/>
+        <Annotation Term="Analytics.AggregatedProperty" Qualifier="sum">
+          <Record>
+            <PropertyValue Property="Name" String="Total"/>
+            <PropertyValue Property="AggregationMethod" String="sum"/>
+            <PropertyValue Property="AggregatableProperty" PropertyPath="Amount"/>
+          </Record>
+        </Annotation>
+        <Annotation Term="Analytics.AggregatedProperty" Qualifier="max">
+          <Record>
+            <PropertyValue Property="Name" String="Maximum"/>
+            <PropertyValue Property="AggregationMethod" String="max"/>
+            <PropertyValue Property="AggregatableProperty" PropertyPath="Amount"/>
+          </Record>
+        </Annotation>
+        <Annotation Term="UI.Chart">
+          <Record>
+            <PropertyValue Property="ChartType" EnumMember="UI.ChartType/Bubble"/>
+            <PropertyValue Property="Dimensions">
+              <Collection>
+                <PropertyPath>Country</PropertyPath>
+              </Collection>
+            </PropertyValue>
+            <PropertyValue Property="Measures">
+              <Collection>
+                <PropertyPath>NumberOfItems</PropertyPath>
+                <PropertyPath>@Analytics.AggregatedProperty#sum</PropertyPath>
```

Comment:
This property path references the complex "property" `@Analytics.AggregatedProperty#sum` of type `Analytics.AggregatedPropertyType` whose value is the constant object
~~~js
 { Name: "Total", AggregationMethod: "sum", AggregatableProperty: "Amount" }
~~~

I would not know how to use this complex type as a measure, given that its "value" is constant on all instances of the "Sales" entity type.

There seem to be some hidden assumptions here on how to extend client interpretation logic for "measures".

Please make these explicit.

Reply:
https://github.com/SAP/odata-vocabularies/blob/2b4eece969576df6ace263fcae47add2f857885a/vocabularies/Analytics.xml#L123-L125
Not explicit enough?

Reply:
Then let's add changes to those annotations that now refer here also in this pull request and check whether the combined documentation is easy to understand.